nucypher/blockchain/eth/sol/source/contracts/MiningAdjudicator.sol

```diff
@@ -125,7 +125,7 @@ contract MiningAdjudicator is Upgradeable {
         address miner = SignatureVerifier.recover(
             SignatureVerifier.hash(_minerPublicKey, hashAlgorithm), _minerPublicKeySignature);
         // Check that miner can be slashed
-        (uint256 minerValue,,,) = escrow.minerInfo(miner);
+        (uint256 minerValue,,,,,) = escrow.minerInfo(miner);
```

Comment:
Woah - what?

Reply:
I added `reStake` parameters to `minerInfo`, so `minerInfo()` returns two parameters more, comas here for ignoring this parameters

Reply:
OK then - that's *very* easy to miss in the future. :sweat_smile: - I wish solidity offered a better way here.

Reply:
Yeah, I missed that first time :) but compiler caught this